CONTRIBUTING.md

```diff
@@ -22,17 +22,44 @@ No software is bug free. So, if you got an issue, follow these steps:
 
 To test your project against the current latest version of storybook, you can clone the repository and link it with `yarn`. Try following these steps:
 
-1.  Download the latest version of this project, and build it:
+#### 1. Download the latest version of this project, and build it:
+
+```sh
+git clone https://github.com/storybooks/storybook.git
+cd storybook
+yarn install
```

Comment:
I think @Hypnosphi opted to use the shorthand `yarn` notation. We should keep it consistent